components/footer/footer.tsx

```diff
@@ -0,0 +1,123 @@
+import Image from 'next/image';
+import styles from '../../styles/Home.module.css';
```

Comment:
This can get ripped out.

Reply:
Will do

Reply:
Do we still need to rip Image out if we're using it for the implementation of the footer image?

Reply:
Ah I meant `styles` not Image. I should have been more clear.

Reply:
npnp -- removed styles